sim/warlock/talents.go

```diff
@@ -277,7 +253,8 @@ func (warlock *Warlock) setupShadowEmbrace() {
 		OnStacksChange: func(aura *core.Aura, sim *core.Simulation, oldStacks int32, newStacks int32) {
 			aura.Unit.PseudoStats.PeriodicShadowDamageDealtMultiplier /= 1.0 + 0.01*float64(warlock.Talents.ShadowEmbrace)*float64(oldStacks)
```

Comment:
The `0.01+float64(warlock.Talents.ShadowEmbrace)` can be precomputed outside this function.